CONTRIBUTING.md

```diff
@@ -0,0 +1,197 @@
+[github-forking-how-to]: https://docs.github.com/en/get-started/quickstart/fork-a-repo
+[installing-git]: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+[installing-npm]: https://nodejs.org/en/download/
+[pro-git]: https://git-scm.com/book/en/v2
+[conda-dot-org-repo]: https://github.com/conda-incubator/conda-dot-org
+[vscode-download]: https://code.visualstudio.com/?wt.mc_id=vscom_downloads
+[sublimetext-download]: https://www.sublimetext.com/3
+[atom-download]: https://atom.io/
+[docusaurus]: https://docusaurus.io/
+[markdown-guide]: https://www.markdownguide.org/
+[issue-board]: https://github.com/orgs/conda-incubator/projects/3
+
+# Contributing
+
+Welcome to the conda.org contributing guide! In this guide, you will find everything you need
+to get up and running as a contributor to conda.org. This is a contributing guide
+for two kinds of contributors:
+
+- **Content contributors**
+    - Contributors writing content for the website, which can include news and blog posts.
+- **Technical contributors**
+    - Contributors writing HTML, CSS, or Javascript for the website that modifies the way 
+      it looks or behaves.
+
+In this guide, we start off by explaining contribution steps that apply
+to both kinds of contributors. Then, the guide branches off to explain specifics for each.
+
+## Documentation tools
+
+conda.org is built with [Docusaurus 2][docusaurus], a modern static website generator that 
+uses a friendly Markdown format to create the website copy. If you are not familiar with 
+Markdown, check out the [Markdown Guide project][markdown-guide].
+
+## Working on issues
+
+The conda.org project has an [issue board][issue-board] from which technical contributors can choose issues. If there is an aspect of the website that you feel needs work, please create a new issue using the **Add item** button.
```

Comment:
Maybe say "using an **Add item** button at the bottom of each column."

I had a hard time finding it.